### ISSUE TITLE
Package cannot be built into a wheel because of 1970 timestamps

### ISSUE THREAD

**Issue by mhsmith** [python-acoustics/python-acoustics#243]
The sdist for version 0.2.4 has many files with a timestamp of Jan 1 1970, which causes pip to fail to build it into a wheel:

```
  creating '/tmp/pip-wheel-rl9m55to/acoustics-0.2.4-cp36-none-any.whl' and adding 'build/bdist.linux-x86_64/wheel' to it
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/tmp/pip-install-uxf97wdv/acoustics/setup.py", line 23, in <module>
      cmdclass={'test': PyTest},
    File "/home/smith/.virtualenvs/pkgtest/lib/python3.6/site-packages/setuptools/__init__.py", line 129, in setup
      return distutils.core.setup(**attrs)
    File "/usr/lib/python3.6/distutils/core.py", line 148, in setup
      dist.run_commands()
    File "/usr/lib/python3.6/distutils/dist.py", line 955, in run_commands
      self.run_command(cmd)
    File "/usr/lib/python3.6/distutils/dist.py", line 974, in run_command
      cmd_obj.run()
    File "/home/smith/.virtualenvs/pkgtest/lib/python3.6/site-packages/wheel/bdist_wheel.py", line 254, in run
      wf.write_files(archive_root)
    File "/home/smith/.virtualenvs/pkgtest/lib/python3.6/site-packages/wheel/wheelfile.py", line 124, in write_files
      self.write(path, arcname)
    File "/home/smith/.virtualenvs/pkgtest/lib/python3.6/site-packages/wheel/wheelfile.py", line 135, in write
      zinfo = ZipInfo(arcname or filename, date_time=get_zipinfo_datetime(st.st_mtime))
    File "/usr/lib/python3.6/zipfile.py", line 338, in __init__
      raise ValueError('ZIP does not support timestamps before 1980')
  ValueError: ZIP does not support timestamps before 1980
  ----------------------------------------
  ERROR: Failed building wheel for acoustics
```

pip then falls back on running `setup.py install`, which succeeds, but the wheel failure slows down the install process and clutters the build log.

This could be fixed by changing the timestamp to something later than Jan 1 1980 (or preferably Jan 2 to be on the safe side, in case of timezone effects).

**Comment by FRidh** [comment]
Haha, did not think of this. I typically use Nix in conjunction with Python, and there we set the timestamps to 1970 for reproducibility. We then handle the issue you mention in the eventual build of the package. In the CI I used Nix as well to create the sdist, thereby producing this issue. I will indeed need to change the date. Thank you for reporting.

**Comment by FRidh** [comment]
fixed in release 0.2.4.post0
